src/internal_events/template.rs

```diff
@@ -16,17 +16,25 @@ impl<'a> InternalEvent for TemplateRenderingError<'a> {
             use std::fmt::Write;
             let _ = write!(msg, " for \"{}\"", field);
         }
+        msg.push('.');
         if self.drop_event {
```

Comment:
I think you should invert if / else blocks because "Events dropped" gets logged when `self.drop_event` is `false`.